MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: Setting Sliding Panel Content in .NET MAUI NavigationDrawer | Syncfusion
+description: Learn here all about Setting Sliding Panel Content support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+
+# Setting Sliding Panel Content in .NET MAUI Navigation Drawer
+
+The drawer pane content is only viewable when the drawer is in the open state. Its content can be set as
+
+*	Header Content
+
+*	Drawer Content
+
+*	Footer Content
+
+N> Header and Footer content are optional but Drawer content is mandatory to allocate space for the drawer.
```

Comment:
N> Header and Footer content is optional, but the Drawer content is mandatory to allocate space for the drawer.

Reply:
Changed